src/components/inputs/InputNumber.js

```diff
@@ -10,19 +13,39 @@ export const InputNumber = ({
     disabled,
     ...props
 }) => {
-    const max = maxValue && maxValue.toString()
+    let key = ''
+
+    switch (keyDownload) {
+        case 'teiDownload':
+            key = 'TEI'
+            break
+        case 'eventsDownload':
+            key = 'events'
+            break
+    }
+
+    const showWarning = value >= maxValue
+    const validationText = showWarning
+        ? i18n.t(
+              'It is recommended to download a maximum of {{maxValue}} {{key}}',
+              { maxValue, key }
+          )
+        : ''
```

Comment:
`TEI` and `events` may be referred to differently in different languages, so I'd suggest generating the validation text like this:

```suggestion
    let warning = ''
    switch (keyDownload) {
        case 'teiDownload':
            warning = i18n.t('Recommended maximum is {{maxValue}} TEI', { maxValue })
            break
        case 'eventsDownload':
            warning = i18n.t('Recommended maximum is {{maxValue}} events', { maxValue })
            break
    }

    const showWarning = value >= maxValue
    const validationText = showWarning ? warning : ''
```